_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1,1420 +1,1661 @@
-/turf/open/floor/plasteel,
+/turf/open/floor/plasteel/white,
 /area/ship/cargo)
-"tO" = (
-/obj/structure/closet/secure_closet{
-	icon_state = "med_secure";
-	name = "medical intern's locker"
+"uT" = (
+/obj/effect/turf_decal/industrial/loading,
+/obj/machinery/defibrillator_mount/loaded{
+	pixel_y = 0;
+	pixel_x = -32
 	},
-/obj/machinery/airalarm/directional/south,
-/obj/item/clothing/glasses/hud/health,
-/obj/item/clothing/glasses/hud/health,
-/obj/item/clothing/glasses/hud/health,
-/obj/item/healthanalyzer,
-/obj/item/healthanalyzer,
-/obj/item/healthanalyzer,
-/obj/item/storage/backpack/satchel/med,
-/obj/item/storage/backpack/satchel/med,
-/obj/item/storage/backpack/satchel/med,
-/obj/item/clothing/shoes/sneakers/blue,
-/obj/item/clothing/shoes/sneakers/blue,
-/obj/item/clothing/shoes/sneakers/blue,
-/obj/structure/extinguisher_cabinet/directional/west,
-/obj/item/clothing/under/rank/medical/paramedic/emt,
-/obj/item/clothing/under/rank/medical/paramedic/emt,
-/obj/item/clothing/under/rank/medical/paramedic/emt,
-/turf/open/floor/plasteel/grimy,
+/obj/structure/sink{
+	dir = 4;
+	pixel_y = 0;
+	pixel_x = -13
+	},
+/turf/open/floor/plasteel/patterned/brushed,
+/area/ship/medical)
+"uU" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/obj/machinery/door/firedoor/border_only{
+	dir = 1
+	},
+/obj/machinery/door/firedoor/border_only,
+/obj/machinery/door/airlock/medical{
+	name = "Storage Room"
+	},
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/storage)
+"vj" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
+	dir = 8
+	},
+/obj/structure/closet/secure_closet/wall{
+	icon_state = "sec_wall";
+	name = "firearms locker";
+	req_access_txt = "5";
+	pixel_y = 28
+	},
+/obj/item/ammo_box/magazine/m10mm/rubber,
+/obj/item/ammo_box/magazine/m10mm/rubber,
+/obj/item/ammo_box/magazine/m10mm/rubber,
+/obj/item/ammo_box/magazine/m10mm/ap,
+/obj/item/ammo_box/magazine/m10mm/ap,
+/obj/item/ammo_box/magazine/m10mm/ap,
+/obj/item/gun/ballistic/automatic/pistol,
+/obj/item/gun/ballistic/automatic/pistol,
+/obj/item/gun/ballistic/automatic/pistol,
+/obj/machinery/light/small/directional/east,
+/turf/open/floor/plasteel/tech,
 /area/ship/crew)
-"tT" = (
-/obj/machinery/light/directional/east,
-/obj/structure/chair{
+"vn" = (
+/obj/machinery/smartfridge/organ{
+	density = 0;
+	pixel_x = 32
+	},
+/obj/structure/table/glass,
+/obj/machinery/cell_charger,
+/turf/open/floor/plasteel/tech,
+/area/ship/medical)
+"vx" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
 	dir = 8
 	},
+/obj/structure/catwalk/over,
+/turf/open/floor/plating,
+/area/ship/external)
+"vE" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 4;
+	name = "Helm"
+	},
+/obj/effect/landmark/start/captain,
+/turf/open/floor/carpet/cyan,
+/area/ship/bridge)
+"wc" = (
+/obj/machinery/door/airlock/maintenance_hatch{
```

Comment:
The floors under these maint hatches connecting the lobby and nacelle tunnels are just plain plasteel. We should probably switch them to some kind of tech floor and/or use some decals to make the area around them less bland.